sysfs.go

```diff
@@ -1,25 +1,40 @@
 package gpio
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"strconv"
 )
 
-type direction uint
+type Direction uint
 
 const (
-	inDirection direction = iota
-	outDirection
+	InDirection Direction = iota
```

Comment:
I'm not sure I understand why these are being made public. It seems even in this PR the user can't access the fields they're on and can't call with them, which IMO makes a strong argument that they should be private